nl/firefox-ios.xliff

```diff
@@ -907,11 +906,6 @@
         <target>Nieuw tabblad geopend</target>
         <note>The label text in the Button Toast for switching to a fresh New Tab.</note>
       </trans-unit>
-      <trans-unit id="Copy Address">
```

Comment:
@st3fan Why was this removed? What is the bug for this one?

Reply:
One is used by the accessibility actions and another by the longpress menu